### PR TITLE
refactor: moved chapi screens outside of the dashboard layout route

### PR DESCRIPTION
**Pull request by birtony**
Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

File: cmd/wallet-web/src/router/routes.js [hunk 1]
```diff
@@ -23,17 +23,12 @@ export default [
         meta: { requiresAuth: true },
       },
       {
-        path: 'logout',
```
Comment (rolsonquadras) [inline cmd/wallet-web/src/router/routes.js:26]: We don't need this path ?

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-web/src/router/routes.js (hunk 1 above)
Comment (birtony) [reply cmd/wallet-web/src/router/routes.js:26]: We shouldn't. It's a component, not a page. We are also using `Signout.vue` now, instead. Let me remove the `Logout.vue` as well

**Review by sudeshrshetty — APPROVED** [review]
make sure CHAPI pages shows trustbloc footers

**Comment by birtony** [comment]
> make sure CHAPI pages shows trustbloc footers

Will add in a follow-up PR

**Comment by rolsonquadras** [comment]
> make sure CHAPI pages shows trustbloc footers

@birtony @sudeshrshetty Can we do this in a follow-up PR ? Currently, our main is broken in sandbox.

**Pull request merged**